metadata.json

```diff
@@ -8,6 +8,14 @@
   "project_page": "https://github.com/voxpupuli/puppet-nrpe",
   "issues_url": "https://github.com/voxpupuli/puppet-nrpe/issues",
   "dependencies": [
+    {
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">= 4.25.1 < 6.0.0"
+    },
+    {
+      "name": "puppetlabs/concat",
+      "version_requirement": ">= 3.0.0 < 6.0.0"
```

Comment:
does version 3 actually work? If I add a dependency I always set the lower version boundary to the current major release.

Reply:
yes, probably.  The current major release is only a couple of months old.